support-frontend/assets/pages/digital-subscription-checkout/components/billingPeriodSelector.jsx

```diff
@@ -0,0 +1,60 @@
+// @flow
+import React from 'react';
+
+import { Fieldset } from 'components/forms/fieldset';
+import { RadioInputWithHelper } from 'components/forms/customFields/radioInputWithHelper';
+import type { DigitalBillingPeriod } from 'helpers/billingPeriods';
+import { billingPeriodTitle } from 'helpers/billingPeriods';
+import { FormSection } from 'components/checkoutForm/checkoutForm';
+import type { ProductPrices } from 'helpers/productPrice/productPrices';
+import { getProductPrice } from 'helpers/productPrice/productPrices';
+import type { IsoCountry } from 'helpers/internationalisation/country';
+import type { Action } from 'helpers/subscriptionsForms/formActions';
+import {
+  getBillingDescription,
+  hasDiscountOrPromotion,
+} from 'helpers/productPrice/priceDescriptionsDigital';
+import {
+  getAppliedPromoDescription,
+} from 'helpers/productPrice/priceDescriptions';
+
+
+type PropTypes = {|
+  productPrices: ProductPrices,
+  billingPeriods: DigitalBillingPeriod[],
+  billingCountry: IsoCountry,
+  selected: DigitalBillingPeriod,
+  onChange: (DigitalBillingPeriod) => Action,
+|}
+
+function BillingPeriodSelectorDigital(props: PropTypes) {
+  return (
+    <FormSection title="How often would you like to pay?">
+      <Fieldset legend="How often would you like to pay?" role="radiogroup">
+        {props.billingPeriods.map((billingPeriod) => {
+          const productPrice = getProductPrice(
+            props.productPrices,
+            props.billingCountry,
+            billingPeriod,
+            'NoFulfilmentOptions',
```

Comment:
It looks like there's an export `NoFulfilmentOptions` from the `fulfilmentOptions` module that we can use instead of the passing the string 'NoFulfilmentOptions'. If the value of `NoFulfilmentOptions` were to ever change it'll save having to make the change in here as well by importing it.

Reply:
We are no longer using this module (for the time being) but that is a good point.